packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -832,17 +832,19 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
 
         if (widget is Text) {
           assert(descendantText == null);
-          final Iterable<Element> matches = find.text(widget.data).evaluate();
+          assert(widget.data != null || widget.textSpan != null);
+          final String text = widget.data ?? widget.textSpan!.toPlainText();
+          final Iterable<Element> matches = find.text(text).evaluate();
           descendantText = widget.data;
           if (matches.length == 1) {
-            debugPrint("  find.text('${widget.data}')");
+            debugPrint("  find.text('$text')");
```

Comment:
i wonder how many people ever use this and whether we should just remove it. I don't think I've used it once since implementing it.

Reply:
(just an idle thought unrelated to this PR obviously)